aries-site/src/data/wcag/components.json

```diff
@@ -2,15 +2,33 @@
   {
     "anchor": [
       { "rule": "1.4.1", "status": "passed" },
-      { "rule": "1.4.6", "status": "passed" },
       { "rule": "1.4.3", "status": "passed" },
+      { "rule": "1.4.6", "status": "passed" },
       { "rule": "2.1.1", "status": "passed" },
       { "rule": "2.1.2", "status": "passed" },
       { "rule": "2.1.3", "status": "passed" },
       { "rule": "2.4.4", "status": "conditional" },
       { "rule": "2.4.7", "status": "passed" },
       { "rule": "2.4.9", "status": "conditional" },
+      { "rule": "2.4.13", "status": "failed" },
+      { "rule": "3.2.1", "status": "passed" },
+      { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.3", "status": "passed" },
+      { "rule": "1.4.6", "status": "passed" },
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
+      { "rule": "2.1.1", "status": "passed" },
+      { "rule": "2.1.2", "status": "passed" },
+      { "rule": "2.4.4", "status": "conditional" },
+      { "rule": "2.4.7", "status": "failed" },
```

Comment:
Created an issue to track this https://github.com/grommet/grommet/issues/7470

Reply:
sounds good thank you

Reply:
@jcfilben - Thank you for the tracking issue.